ote_sdk/ote_sdk/entities/metrics.py

```diff
@@ -702,3 +707,34 @@ def __repr__(self):
 
     def __eq__(self, other):
         return isinstance(other, NullPerformance)
+
+
+class AnomalyLocalizationPerformance(Performance):
```

Comment:
We need this class to support multi-metric projects. We initially looked at generic MultiScorePerformance implementations, but this is tricky because it would be hard to distinguish the different metrics from each other, especially if not all metrics can be reported at all times (like in the example above with only global, no local annotations). Since anomaly tasks are currently the only tasks that require multi-metric support, we decided to make this class specific to anomaly.

Reply:
I refactored the class so that we now have a generic MultiScorePerformance class that supports adding one or more additional metrics, and a subclass AnomalyLocalizationPerformance which contains the global and local scores specific for anomaly tasks.